gravitee-apim-console-webui/src/management/integrations/integration-status/integration-status.component.scss

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.icon {
+  margin-right: 5px;
```

Comment:
not needed : https://particles.gravitee.io/?path=/docs/components-badge-readme--docs 
you can use `class="gio-left"` 

--
however, more globally. we don't add margin, padding, any space which are not multiples of 2 (2, 4, 6, 8)
((except when it's a border and padding add-on, for example))
This makes the calculation much simpler. 
It's also a rule that the UI/ UX team respects. We set it together quite a few years ago.

Reply:
changed